src/Rifm.js

```diff
@@ -100,6 +100,15 @@ export const useRifm = (props: Args): RenderProps => {
       // in case of isDeleleteButtonDown cursor should move differently vs backspace
       const deleteWasNoOp = isDeleleteButtonDown && isNoOperation;
 
+      const valueAfterSelectionStart = eventValue.slice(input.selectionStart);
+
+      const acceptedCharIndexAfterDelete = valueAfterSelectionStart.search(
+        props.accept || /\d/g
+      );
+
+      const charsToSkipAfterDelete =
```

Comment:
just acceptedCharIndexAfterDelete + 1 no need in in ifs here
having that below you can just use charsToSkipAfterDelete without +1

Reply:
? probably needed as change above changes current logic, but probably will work even without